doc/topics/reactor/index.rst

```diff
@@ -219,27 +423,52 @@ For example, to access the ``id`` field of the beacon event in a reactor file,
 you will need to reference ``{{ data['data']['id'] }}`` rather than ``{{
 data['id'] }}`` as for events initiated directly on the event bus.
 
+Similarly, the data dictionary attached to the event's would be located in
```

Comment:
the possessive form of `event` is confusing here imho.

Reply:
The `'s` shouldn't have been there, I've removed it.